pkg/apis/kops/networking.go

```diff
@@ -124,6 +124,11 @@ type RomanaNetworkingSpec struct {
 
 // AmazonVPCNetworkingSpec declares that we want Amazon VPC CNI networking
 type AmazonVPCNetworkingSpec struct {
+	// Allow passing the vethprefix to work with calico networking policies
+	// https://aws.amazon.com/blogs/opensource/networking-foundation-eks-aws-cni-calico
+	Vethprefix string `json:"vethprefix,omitempty"`
```

Comment:
I _think_ the JSON should probably be `vethPrefix`.  Or maybe `interfacePrefix` ?